### DISCUSSION TITLE
Screen Checks not working?

### DISCUSSION THREAD

**Discussion by ollowain149** [Lailloken/Exile-UI#139] [General]
Hi,
just found this tool and wanted to set up everything. But I cannot figure out how to do the screen checks e.g. for Betrayal.
I'm pressing calibrate, try to do the exact screenshot as described, the screenshot is taken by windows snipping tool and put to clipboard (pop up on bottom right says so). But still no test is positive.
When I try to open the img folder I get an error message.
![image](https://user-images.githubusercontent.com/120138030/206566344-dc2f259c-5423-455f-a544-f0d84b8dd1aa.png)Any ideas to fix?

**Comment by Lailloken** [comment]
Could you check the img folder (Edit: manually, by opening the script folder) and tell me what's inside it? There should be a '_Fallback', 'Betrayal', 'GUI', and a 'Recognition (900p)' folder. If the recognition folder does not exist, you need to create it manually and also a 'Betrayal' and 'GUI' folder inside it.

Someone else had the same problem with missing folders not too long ago, and I thought I fixed it. There seems to be a problem with write permissions under certain circumstances.

**Comment by ollowain149** [comment]
Thanks for the fast reply. Was acutally due to admin rights. Fixed it, not everything works.

**Reply by Lailloken** [reply]
What did you change? There are reports popping up right now from people also having write-permission issues.
